contrib/otlp/Exporter.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\otlp;
+
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
+use GuzzleHttp\Psr7\Request;
+use Http\Adapter\Guzzle6\Client;
+use Psr\Http\Client\ClientExceptionInterface;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Client\NetworkExceptionInterface;
+use Psr\Http\Client\RequestExceptionInterface;
+use OpenTelemetry\Sdk\Trace;
+use OpenTelemetry\Trace as API;
+
+
+class Exporter implements Trace\Exporter
+{
+    /**
+     * @var string
+     */
+     private $endpointURL;
+
+    /**
+     * @var string
+     */
+    private $protocol;
+
+    /**
+     * @var string
+     */
+    private $insecure;
+
+    /**
+     * @var string
+     */
+    private $certificateFile;
+
+    /**
+     * @var array
+     */
+    private $headers;
+
+    /**
+     * @var string
+     */
+    private $compression;
+
+    /**
+     * @var int
+     */
+    private $timeout;
+    /**
+     * @var SpanConverter
+     */
+    private $spanConverter;
+    
+     /**
+     * @var bool
+     */
+    private $running = true;
+
+    /**
+     * @var ClientInterface
+     */
+
+    private $client;
+
+    /**
+     * Exporter constructor.
+     * @param string $serviceName 
+     */
+    public function __construct(
+        $serviceName
+    )
+    {
+
+        // Set default values based on presence of env variable
+        $this->endpointURL = getenv("OTEL_EXPORTER_OTLP_ENDPOINT") ?: "localhost:55680";
+        $this->protocol = getenv("OTEL_EXPORTER_OTLP_PROTOCOL") ?: "grpc";
+        $this->insecure = getenv("OTEL_EXPORTER_OTLP_INSECURE") ?: "false";
+        $this->certificateFile = getenv("OTEL_EXPORTER_OTLP_CERTIFICATE") ?: "none";
+        $this->headers[] = getenv("OTEL_EXPORTER_OTLP_HEADERS") ?: "none";
+        $this->compression = getenv("OTEL_EXPORTER_OTLP_COMPRESSION") ?: "none";
+        $this->timeout = getenv("OTEL_EXPORTER_OTLP_TIMEOUT") ?: 10;
```

Comment:
`?:` is unsafe here since it will fail with `0`.

https://3v4l.org/lmWjD

Reply:
You're free to change this - it's your PR 👍

Reply:
Eh, it might be a case of mistaken identity here. :)

Reply:
Sorry, was referring to @riticksingh - I could see how that would be confusing!

Reply:
NP :+1: